llvm/lib/Transforms/Vectorize/VPlanValue.h

```diff
@@ -197,6 +176,51 @@ class LLVM_ABI_FOR_TEST VPValue {
 LLVM_ABI_FOR_TEST raw_ostream &operator<<(raw_ostream &OS,
                                           const VPRecipeBase &R);
 
+/// A VPValue representing a live-in from the input IR. It wraps an underlying
+/// IR Value.
```

Comment:
```suggestion
/// A VPValue wrapping an underlying IR Value.
```
May be updated when existing live-in IR Values are distinguished from newly created constants, etc.